kolibri/plugins/learn/assets/src/views/search-page/index.vue

```diff
@@ -61,18 +61,19 @@
 
       <p v-if="filteredResults.length === 0">{{ noResultsMsg }}</p>
 
-      <card-grid v-else>
-        <content-grid-item
-          v-for="content in contents"
-          v-show="filter === 'all' || filter === content.kind"
-          :key="content.id"
-          :title="content.title"
-          :thumbnail="content.thumbnail"
-          :progress="content.progress"
-          :kind="content.kind"
-          :link="genLink(content)"
-        />
-      </card-grid>
+      <content-card-grid v-else :contents="filteredResults">
+        <template scope="content">
```

Comment:
in hindsight, a scoped slot was probably overkill for the grid in its current state. I think I might just switch to a normal slot tomorrow.